### DISCUSSION TITLE
Can Excel or CSV report be generated?

### DISCUSSION THREAD

**Discussion by Vibi-007** [sinaptik-ai/pandas-ai#1124] [Q&A]
Is it possible to generate.xlsx or .csv report and save it since in pandas we have .to_excel or .to_csv method.

**Comment by junle-chen** [comment]
Yes, you can add this sentence at the end of the prompt: please save the file in csv format.

**Reply by Vibi-007** [reply]
I have tried it and it successfully shows 'file saved as output.csv' but where it is actually stored i am using vertexai environment and also when I checked  the last_code_executed it seems to be passed just as string rather than using .to_csv conversion method.
